src/harness/testcases/WINNF_FT_S_SCS_testcase.py

```diff
@@ -64,3 +67,83 @@ def test_WINNF_FT_S_SCS_5(self):
     Checks that a CBSD registration with this configuration succeed.
     """
     self.doTestCipher('ECDHE-RSA-AES128-GCM-SHA256')
+
+
+
+  @winnforum_testcase
+  def test_WINNF_FT_S_SCS_12(self):
+    """Expired certificate presented during registration.
+    Checks that SAS UUT response with fatal alert message.
+    """
+    device_cert = self.getCertFilename('client_expired.cert')
```

Comment:
Should this be configurable?

Reply:
Yes.   It is good to have these negative test cases as configurable, especially the   ones that use a certificate that is different from the default 'client.cert'.   This will allow different certificate to be used during ITS testing without   modifying the test case.      The cert path are configurable now and added into      default.config for tests SCS12 and SCS15.          Please note that the only the client.cert and client.key are made   configurable. The CA Cert file will be the default one that is used in the   sas.py file. If a different root of trust has to be used then this file has   to be changed. This is however applicable to the existing positive test cases   as well (SCS1 to 5).